TeachingRecordSystem/src/TeachingRecordSystem.Core/DataStore/Postgres/Models/Person.cs

```diff
@@ -13,10 +20,11 @@ public class Person
     public DateTime? DeletedOn { get; set; }
     public required string? Trn { get; set; }
     public required string FirstName { get; set; }
-    public required string MiddleName { get; set; }
+    public required string? MiddleName { get; set; }
     public required string LastName { get; set; }
     public required DateOnly? DateOfBirth { get; set; }  // A few DQT records in prod have a null DOB
     public string? EmailAddress { get; set; }
+    public string? MobileNumber { get; set; }
```

Comment:
We'll need a migration adding for the reporting database too - `TeachingRecordSystem.Core.Services.DqtReporting.Migrations`

Reply:
@gunndabad like this?
https://github.com/DFE-Digital/teaching-record-system/pull/2022/files#diff-0c8c8eeb88a73c65d65e6993632a31fa089d85d92070113b3988ba29732c3c98R1